core/src/test/java/com/graphhopper/routing/util/parsers/MountainBikeTagParserTest.java

```diff
@@ -64,35 +61,35 @@ protected BikeCommonPriorityParser createPriorityParser(EncodedValueLookup looku
     public void testSpeedAndPriority() {
         ReaderWay way = new ReaderWay(1);
         way.setTag("highway", "primary");
-        assertPriorityAndSpeed(BAD, 18, way);
+        assertPriorityAndSpeed(0.5, 18, way);
 
         way.setTag("highway", "residential");
-        assertPriorityAndSpeed(PREFER, 18, way);
+        assertPriorityAndSpeed(1.2, 18, way);
 
         // Test pushing section speeds
         way.setTag("highway", "footway");
-        assertPriorityAndSpeed(SLIGHT_AVOID, 6, way);
+        assertPriorityAndSpeed(0.9, 6, way);
 
         way.setTag("highway", "track");
-        assertPriorityAndSpeed(PREFER, 12, way);
+        assertPriorityAndSpeed(1.2, 12, way);
 
         way.setTag("bicycle", "yes");
-        assertPriorityAndSpeed(PREFER, 12, way);
+        assertPriorityAndSpeed(1.2, 12, way);
 
         way.setTag("highway", "track");
         way.setTag("bicycle", "yes");
         way.setTag("tracktype", "grade3");
-        assertPriorityAndSpeed(VERY_NICE, 12, way);
+        assertPriorityAndSpeed(1.2, 12, way);
```

Comment:
See my comment for the respective code change. I would prefer to keep the old classification here.